src/components/sbb-header/sbb-header.scss

```diff
@@ -33,13 +34,23 @@
   justify-content: flex-start;
   padding: 0;
   margin-inline: auto;
-  height: var(--sbb-header-height-override, var(--sbb-header-height));
+  height: var(--sbb-header-height);
   width: 100%;
-  max-width: #{px-to-rem-build(1200)};
+  max-width: var(--sbb-layout-base-page-max-width);
 }
 
 .sbb-header__logo {
-  height: var(--sbb-logo-height);
-  aspect-ratio: 19/2;
   margin-inline-start: auto;
+
+  // Only apply height to standard logo. If slotted, no styles will be applied.
+  sbb-logo {
+    height: var(--sbb-header-logo-height);
+    aspect-ratio: 19/2;
+  }
+}
+
+// Fix left offset if first element of the header is a sbb-header-action.
+// The value of the offset is calculated inside sbb-header-action styles.
+.sbb-header__wrapper > ::slotted(sbb-header-action:first-child) {
```

Comment:
Should the same be applied to the last child?

Reply:
I think not because the end is the logo and the log doesn't have any margin so it's aligned correctly.